core/app/models/spree/order.rb

```diff
@@ -70,6 +70,11 @@ def generate_number(options = {})
     has_many :products, through: :variants
     has_many :variants, through: :line_items
     has_many :refunds, through: :payments
+    has_many :all_adjustments,
+      class_name: 'Spree::Adjustment',
```

Comment:
I do not like this style. But I want to get this patch in. So I changed to this recommendation.